portability-worker/src/main/java/org/dataportabilityproject/worker/WorkerImpl.java

```diff
@@ -82,23 +88,30 @@ private void processJob(PortabilityJob job) {
     PortableDataType dataType = PortableDataType.valueOf(job.dataType());
     try {
       try {
+        // TODO: Enable worker-based copying
         if(true) {
-          logger.info("Exiting before copy, job: {}", job);
-          // TODO: Enable worker-based copying
+          logger.warn("Exiting before copy, job: {}", job);
```

Comment:
Sorry if I missed this, could you explain why we cant enable this yet? iiuc, this causes us to skip the use of PortabilityCopier below right?

Reply:
Done. It was mostly for debugging easily.